sink/sink

```diff
@@ -89,9 +97,38 @@ def expand(val, env):
     else:
         return val
 
+# Compare two JSON style values to see if one is contained in the other
+# For lists and dict make sure all items in expect are also present in result
+# For any other values a strict comparison is used
+def contains(expect, result, debug=True):
+    if isinstance(expect, list) and isinstance(result, list):
+        result = list(result)
+        for exp in expect:
+            while len(result):
+                res = result.pop(0)
```

Comment:
I've moved back to the old code since this was a step backwards. The intent is to ensure that exactly one of the expected item is contained in the outer container. 

Please see accompanying pull requests in cockpit for cases where this is used.

> Can we somehow avoid the case that multiple identical entries in a list are significant? Perhaps by always appending the container name as a suffix to action labels, or something such?

It's hard. Because the entire point of this is to detect collisions and duplicates.